ffi/cbindgen.toml

```diff
@@ -93,7 +97,7 @@ rename_variants = "None"
 # deprecated = "DEPRECATED_ENUM"
 # deprecated_with_note = "DEPRECATED_ENUM_WITH_NOTE"
 add_sentinel = false
-prefix_with_name = false
+prefix_with_name = true
```

Comment:
this change doesn't affect anything now but does affect a future PR. I didn't mean to include it here but also don't want to spatch out this one line.